openstack/cortex/templates/seed.yaml

```
@@ -33,3 +33,5 @@ spec:
         role: cloud_volume_admin
       - user: scheduler_service@Default
         role: cloud_network_admin
+      - user: scheduler_service@Default
+        role: cloud_sharedfilesystem_admin
```

```suggestion
        role: cloud_sharedfilesystem_viewer
```

is sufficient for initial placement